examples/example-gateway/config/production.yaml

```diff
@@ -3,28 +3,44 @@ clients.bar.defaultHeaders:
 clients.bar.ip: 127.0.0.1
 clients.bar.port: 4001
 clients.bar.timeout: 10000
+clients.bar.maxConcurrentRequests: 1000
+clients.bar.errorPercentThreshold: 20
 clients.baz.ip: 127.0.0.1
 clients.baz.port: 4002
 clients.baz.serviceName: Qux
 clients.baz.timeout: 10000
 clients.baz.timeoutPerAttempt: 2000
+clients.baz.maxConcurrentRequests: 1000
+clients.baz.errorPercentThreshold: 20
 clients.contacts.ip: 127.0.0.1
 clients.contacts.port: 4000
 clients.contacts.timeout: 10000
+clients.contacts.maxConcurrentRequests: 1000
+clients.contacts.errorPercentThreshold: 20
 clients.corge-http.serviceName: CorgeHttp
+clients.corge-http.maxConcurrentRequests: 1000
+clients.corge-http.errorPercentThreshold: 20
 clients.corge.serviceName: Corge
 clients.corge.timeout: 10000
 clients.corge.timeoutPerAttempt: 2000
+clients.corge.maxConcurrentRequests: 1000
+clients.corge.errorPercentThreshold: 20
 clients.google-now.ip: 127.0.0.1
 clients.google-now.port: 14120
 clients.google-now.timeout: 10000
+clients.google-now.maxConcurrentRequests: 1000
```

Comment:
Let's put the disabled config here as well, to be consistent with the json config. (yaml is prefered, but zanzibar is backward compatible with json config)